kibana/utils

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+#
+# Copyright 2018 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#override config by environment variables
+#inspired by https://github.com/elastic/kibana-docker/blob/master/build/kibana/bin/kibana-docker
+
+update_config_from_env_vars() {
+  conf_dir=$1
+  kibana_vars=(
```

Comment:
do we want to let all of these be configured with env vars? e.g. `server.host` may be alright to keep as `0.0.0.0` in the config... is there a request that we be able to customize this via env vars?

Reply:
This is open to debate; I sourced what is here from the script that is used by the Kibana docker image.  I know we have an outstanding issue where the request timeout needs to be increased but the work around is complicated.  This change would make that easy.  I am open to whitelisting the config items to a smaller set.